### PR TITLE
Create Bulletin du Centre d’études médiévales d’Auxerre

### PR DESCRIPTION
**Pull request by FrancoisC37**
Hi,
I've done this style.
I hope it will be fine.

### CODE REVIEW

File: bulletin-du-centre-detudes-medievales-dauxerre.csl [hunk 1]
```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="fr-FR">
+  <info>
+    <title>Bulletin du Centre d’études médiévales d’Auxerre (French)</title>
+    <title-short>Bucema</title-short>
+    <id>http://www.zotero.org/styles/bulletin-du-centre-detudes-medievales-dauxerre</id>
+    <link href="http://www.zotero.org/styles/bulletin-du-centre-detudes-medievales-dauxerre" rel="self"/>
+    <link href="http://www.zotero.org/styles/racf" rel="template"/>
```
Comment (rmzelle) [inline bulletin-du-centre-detudes-medievales-dauxerre.csl]: @FrancoisC37, you list http://www.zotero.org/styles/racf as the template style, but we don't have this style in our repository. Do you know if the Bucema style is derived from a style from our collection (https://github.com/citation-style-language/styles/)?

Comment (FrancoisC37) [inline bulletin-du-centre-detudes-medievales-dauxerre.csl]: http://www.zotero.org/styles/revue-archeologique-du-centre-de-la-france

Comment (FrancoisC37) [reply bulletin-du-centre-detudes-medievales-dauxerre.csl]: It's a mistake...
The style is : http://www.zotero.org/styles/revue-archeologique-du-centre-de-la-france

**Comment by rmzelle** [comment]
Merci beaucoup!

**Pull request merged**